packages/functions/src/list-budgets/index.ts

```diff
@@ -0,0 +1,10 @@
+import { APIGatewayProxyEvent } from "aws-lambda";
+
+export async function handler(event: APIGatewayProxyEvent){
+    return {
+        statusCode: 200,
+        body: JSON.stringify({
+            "budgets": ["foo", "bar"],
```

Comment:
I think API-G will return a `502` if we return a response that doesn't match the schema:
* https://github.com/coltonneil/weekly-budgeter-v2/blob/main/model/weekly-budgeter-v2.json#L719

Reply:
Unfortunately it doesn't look to be enforcing the schema. Looks like we'll need to do output validation.

![image](https://github.com/coltonneil/weekly-budgeter-v2/assets/15051149/d8903fbd-c708-4553-a0f6-5200a397daca)